test/stressLinear.cxx

```diff
@@ -807,7 +807,7 @@ void mstress_binary_ebe_op(Int_t rsize, Int_t csize)
   ok &= VerifyMatrixValue(m1,pattern,gVerbose,EPSILON);
   if (gVerbose)
     std::cout << "  subtracting the matrix from itself" << std::endl;
-  m1 -= m1;
+  m1 = m1 - m1; //avoiding compiler warning
```

Comment:
This is not the same (`op-=` versus `op-` and `op=`), and this shouldn't be changed. We should instead suppress the warning.

Reply:
I will add this in separate PR, after I will verify it is not a problem of compiler.